index.rst

```diff
@@ -5,17 +5,21 @@ Welcome to the official documentation of the UBports project!
 
 UBports develops the mobile phone operating system Ubuntu Touch. Ubuntu Touch is a mobile operating system focused on ease of use, privacy, and convergence.
 
-On this website you find instructions how to install Ubuntu Touch on your mobile phone, user guides and detailed documentation on all system components. If this is your first time here, please consider reading our :doc:`introduction <about/introduction>`.
+On this website you find instructions how to install Ubuntu Touch on your mobile phone, user guides and detailed documentation on all system components. If this is your first time here, please consider reading our :doc:`introduction <general/about_ubuntu_touch>`.
 
 .. note::
     This documentation is currently in a quite volatile state, so don't be alarmed if pages are shuffled around from the last time you were here! If you want to help improving the docs, :doc:`this <contribute/documentation>` will get you started.
 
 .. toctree::
-   :maxdepth: 1
-   :caption: About
-   :name: sec-about
-
-   about/index
+   :maxdepth: 5
```

Comment:
I'd strongly consider changing this to be a depth of 1 to match the below trees. If the goal is to allow people to see more of the subsections, they can do that by expanding a section in the sidebar.

Reply:
This is a bad design decision. Why would you have a general section if it only has one link? And how would you call that link? Introduction (as it is now) is a bad name. Every link in navigation should say what information it gives you. Introduction doesn't say anything. How I would know it will have sub-pages about community? about devices? about Ubuntu Touch in general? This is a documentation and you should not be afraid to scroll. When you go to documentation you except there would be a lot of information. We just need to have a good structure. And putting everything inside one link hides that structure at all.

Reply:
About UBports > Introduction > About UBports ? Is that what we are talking about?

Reply:
Yes